### DISCUSSION TITLE
iOS Build Problems

### DISCUSSION THREAD

**Discussion by Luminarcy** [Poing-Studios/Godot-AdMob-Android-iOS#50] [Q&A] [answered]
I'm trying to build iOS version of my game with AdMob enabled and i have following problem.
![vmware_GNqhS9dqqf](https://user-images.githubusercontent.com/42573485/112267857-20945e00-8c87-11eb-873d-e8b46d55cb26.png)

That's how my Link Binary with Libraries tab looks like
![vmware_6tHrGLKGI9](https://user-images.githubusercontent.com/42573485/112267948-47529480-8c87-11eb-9f96-eae798634b36.png)

Thanks for the help!

**Comment by Luminarcy** [comment]
@gustavottc also, are there any way to build game with AdMob enabled with XCode 11.4 in MacOS Catalina?

**Reply by gumaciel** [reply]
I think yes, but i strong recommend you to update, due of this: https://developers.google.com/admob/ios/ios14

If you dont do that, this will happen: 
![Captura de Tela 2021-03-24 às 08 32 36](https://user-images.githubusercontent.com/20030153/112303877-92ff4100-8c7b-11eb-93b6-592baccefa0f.png)

https://support.google.com/admob/answer/9997589?hl=en

**Comment by gumaciel** [comment, marked as answer]
@Luminarcy if some error of "framework not found" then: [do that](https://user-images.githubusercontent.com/20030153/103795860-45cce500-5025-11eb-87ac-18ef6c6c85c2.png) and [this](https://user-images.githubusercontent.com/20030153/103795850-42395e00-5025-11eb-88e8-c7ff413c7434.png) 
![Captura de Tela 2021-01-17 às 11 29 14](https://user-images.githubusercontent.com/20030153/104846213-28ff9f80-58b8-11eb-902d-c7a704c2baa5.png)


Ps: Ignore the ```libadmob_plugin.a```

**Reply by Luminarcy** [reply]
@gustavottc Thank you! That works.
